source/js/components/expander.js

```diff
@@ -20,17 +20,23 @@ var Panel = React.createClass({
     this.props.activateKey(itemKey);
   },
   render: function() {
+    var expanded = this.props.itemKey === this.props.activeKey;
+    var style = {
+      height: "0"
+    };
     var itemClassName = classnames("expander-item", {
-      "expander-item-active": this.props.itemKey === this.props.activeKey
-    });
-    var contentClassName = classnames("expander-content", {
-      "expander-content-active": this.props.itemKey === this.props.activeKey
+      "expander-item-active": expanded
     });
+    if (this.content && expanded) {
```

Comment:
let's check for `this.props.children` instead - we don't need to create an extra `<div>` if `this.props.children` is null